com.unity.formats.fbx/Tests/FbxTests/FbxConstraintTest.cs

```diff
@@ -23,18 +23,21 @@ public static IEnumerable TestCases
                 yield return new TestCaseData(typeof(RotationConstraint), new float[] { 1.2f, 0.8f, 10.3f }, new float[] { 194, 9, 195 }, "m_RotationOffset.z").Returns(1);
 
                 /* Test Aim */
-                yield return new TestCaseData(typeof(AimConstraint), new float[] { 1f, 30f, 10f }, new float[] { 10f, 180f, 10f }, "m_AimVector.x").Returns(1);
+                yield return new TestCaseData(typeof(AimConstraint), new float[] { 1f, 30f, 10f }, new float[] { 10f, 180f, 10f }, "m_AimVector.x").Returns(1)
+                    .Ignore("UT-3734: Test fails due to constraint import issues.");
```

Comment:
Can you add the fogbugz number as well: 1205373
(either in the text or in a comment next to this line)